sdk/java/src/main/java/sawtooth/sdk/client/Signing.java

```diff
@@ -22,7 +36,7 @@ public static ECKey generatePrivateKey(SecureRandom random) {
   }
 
   public static String getPublicKey(ECKey privateKey) {
-    return "03" + privateKey.getPublicKeyAsHex();
+    return privateKey.getPublicKeyAsHex();
```

Comment:
Hi, I'm the one who mentioned this change in the sawtooth rocketchat. Problem was that the added "03" caused the validator to not be able to verify the signature. Removing it allowed the signature to be verified successfully.